Src/PerformanceCollector/Shared/QuickPulseTelemetryProcessor.cs

```diff
@@ -24,6 +24,12 @@ public class QuickPulseTelemetryProcessor : ITelemetryProcessor, ITelemetryModul
 
         private const int InitialTelemetryQuota = 3;
 
+        private const int MaxFieldLength = 32768;
```

Comment:
>MaxFieldLength [](start = 26, length = 14)

You may want to start thinking about moving sanitization logic in a separate class